Uni_life_tracker/app/src/main/java/com/generic/plannr/Controllers/EmailValidator.java

```diff
@@ -0,0 +1,36 @@
+package com.generic.plannr.Controllers;
+
+import android.util.Patterns;
+
+import com.generic.plannr.Gateways.UserGateway;
+import com.google.android.material.textfield.TextInputLayout;
+
+import java.util.Objects;
+
+public class EmailValidator implements Validator {
+
+
```

Comment:
I think there used to be javadocs for this class if you could add them back in that would be great

Reply:
done